src/components/curation/ClinicalPhenotype.vue

```diff
@@ -83,69 +206,21 @@ export default {
                   :for="`phenotype-hpo-terms-input-${pmid}`"
                   class="col-form-label"
                 >
-                  HPO Terms(s)
+                  HPO Term(s)
                 </label>
               </div>
               <div class="col-4">
                 <textarea
-                  :class="
-                    hpoTermsInputHelper[pmid].isHpoTermsValid
-                      ? 'form-control'
-                      : 'form-control is-invalid'
-                  "
+                  :class="{
+                    'form-control': true,
+                    'is-invalid': !hpoTermsInputHelper[pmid].isHpoTermsValid,
+                  }"
                   :id="`phenotype-hpo-terms-input-${pmid}`"
-                  :value="hpoTermsInputHelper[pmid].hpoTermsInput"
+                  v-model="hpoTermsInputHelper[pmid].hpoTermsInput"
```

Comment:
```suggestion
                  :value="hpoTermsInputHelper[pmid].hpoTermsInput"
```

Reply:
To update the prop clinicalPhenotype, v-model is already used for the prop in the parent component and in this component combination of :value and @input should be used as per the vue documentation (https://vuejs.org/guide/components/v-model.html#v-model-arguments)